crates/biome_js_analyze/tests/specs/nursery/useSemanticElements/invalid.js

```diff
@@ -0,0 +1,11 @@
+const checkbox = () => (
+    <div role="checkbox" ></div>
+);
+
+const Image = () => (
+    <div role="img" ></div>
+);
+
+const Button = () => (
+    <div aria-label="foo" role="button" >button</div>
+);
```

Comment:
```suggestion
<>
    <div role="checkbox" ></div>
    <div role="img" ></div>
    <div aria-label="foo" role="button" >button</div>
</>
```

Way less code. Also, this file should be called `invalid.jsx`, not `invalid.js`.

Also, we should be more thorough with testing, especially with a11y. This means that we should add more tests. Ideally we should cover all roles contained here:

https://github.com/biomejs/biome/blob/9a05b77cbde0e29135783de2ee1450d3dd5a69ea/crates/biome_aria/src/roles.rs#L1185-L1239

It's a lot, so I suggest you defer it to another PR.